openapi/api.github.com/index.json

```diff
@@ -1378,27 +1378,37 @@
       "delete": { "$ref": "operations/teams/remove-member-legacy.json" }
     },
     "/teams/{team_id}/memberships/{username}": {
-      "get": { "$ref": "operations/teams/get-membership-legacy.json" },
+      "get": { "$ref": "operations/teams/get-membership-for-user-legacy.json" },
       "put": {
-        "$ref": "operations/teams/add-or-update-membership-legacy.json"
+        "$ref": "operations/teams/add-or-update-membership-for-user-legacy.json"
       },
-      "delete": { "$ref": "operations/teams/remove-membership-legacy.json" }
+      "delete": {
+        "$ref": "operations/teams/remove-membership-for-user-legacy.json"
+      }
     },
     "/teams/{team_id}/projects": {
       "get": { "$ref": "operations/teams/list-projects-legacy.json" }
     },
     "/teams/{team_id}/projects/{project_id}": {
-      "get": { "$ref": "operations/teams/review-project-legacy.json" },
-      "put": { "$ref": "operations/teams/add-or-update-project-legacy.json" },
-      "delete": { "$ref": "operations/teams/remove-project-legacy.json" }
+      "get": {
+        "$ref": "operations/teams/check-permissions-for-project-legacy.json"
+      },
+      "put": {
+        "$ref": "operations/teams/add-or-update-project-permissions-legacy.json"
+      },
+      "delete": { "$ref": "operations/teams/remove-project-from-legacy.json" }
```

Comment:
needs 3 deprecations

- [ ] `GET /teams/{team_id}/projects/{project_id}`
- [ ] `PUT /teams/{team_id}/projects/{project_id}`
- [ ] `DELETE /teams/{team_id}/projects/{project_id}`. Also needs fix for operation ID. `teams/remove-project-from-legacy` should be `teams/remove-project-from-project-legacy`